bin/hhast-dump.hack

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env hhvm
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST\__Private;
+
+<<__EntryPoint>>
+async function hhast_dump_main_async(): Awaitable<noreturn> {
+  $root = \realpath(__DIR__.'/..');
+  $found_autoloader = false;
+  while (true) {
+    $autoloader = $root.'/vendor/autoload.hack';
+    if (\file_exists($autoloader)) {
+      $found_autoloader = true;
+      require_once($autoloader);
```

Comment:
What is autoloader?

Reply:
"You want class Foo? It's in src/foo.hack. Function bar? src/bar.hack"

An autoloader is required for all hack programs that are not single-file.